pack/tic/itd_encounter.json

```diff
@@ -1,19 +1,531 @@
+		"back_name": "Closing In",
+		"back_text": "Increase the flood level of each location. \nIn player order, each investigator draws 1 encounter card.",
+		"code": "07126",
+		"doom": 4,
+		"double_sided": true,
+		"encounter_code": "in_too_deep",
+		"encounter_position": 4,
+		"faction_code": "mythos",
+		"flavor": "\"As I watched - choked by a sudden rise in the fishy odour after a short abatement - I saw a band of uncouth, crouching shapes loping and shambling in the same direction...\"\n - H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"illustrator": "Matt Dixon",
+		"name": "Flooded Streets",
+		"pack_code": "itd",
+		"position": 126,
+		"quantity": 1,
+		"stage": 3,
+		"text": "Each [[Suspect]] enemy loses aloof and cannot be parleyed with.\n<b>Forced</b> - When your turn ends, if you are at a fully flooded location: Take 1 damage, 1 horror, and discard 1 card at random from your hand.",
+		"type_code": "agenda"
+	},
+	{
+		"back_flavor": "Water rushes through the town. The waves crash through dilapidated buildings, consume the streets, and destroy every standing bridge. Even the refinery is turned to rubble by the surging waters. Soon, Innsmouth is gone. You barely escape the raging sea.",
+		"back_name": "Swallowed by the Sea",
+		"back_text": "In your Campaign Log, record <i>Innsmouth was consumed by the rising tide.</i>\nEach investigator that has not resigned is defeated and suffers 1 physical trauma.",
+		"code": "07127",
+		"doom": 3,
+		"double_sided": true,
+		"encounter_code": "in_too_deep",
+		"encounter_position": 5,
+		"faction_code": "mythos",
+		"flavor": "\"Nothing that I could have imagined - nothing, even, that I could have gathered had I credited old Zadok's crazy tale in the most literal way - would be in any way comparable to the demoniac, blasphemous reality that I saw - or believe I saw.\"\n - H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"illustrator": "Matt Dixon",
+		"name": "Rage of the Deep",
+		"pack_code": "itd",
+		"position": 127,
+		"quantity": 1,
+		"stage": 4,
+		"text": "Each [[Suspect]] enemy loses aloof and cannot be parleyed with.\n<b>Forced</b> - When your turn ends, if you are at a fully flooded location: Take 1 damage, 1 horror, and discard 2 cards at random from your hand.",
+		"type_code": "agenda"
+	},
+	{
+		"back_flavor": "After hours of fleeing, sneaking, and hiding, you finally reach the abandoned railway station on the western edge of Innsmouth. The moment you see the two automobiles parked by the tracks, you break into a run. You're more than happy to leave this awful place behind.",
+		"back_name": "Out of Town",
+		"back_text": "\nIn your Campaign Log, record <b>(&rarr;R1)</b>",
```

Comment:
There is a little bit missing here. After record, in italics (with the italic tag), please add 'the investigators made it safely to their vehicles'. 

For the R1 section, please delete '&rarr;' and insert '→'